examples/split.js

```diff
@@ -2,7 +2,7 @@ const { program } = require('commander');
 
 // This is used as an example in the README for the Quick Start.
 
-program.option('--first').option('-s, --separator <char>');
+program.option('--first').option('-s, --separator <char>').argument('<string>');
```

Comment:
```suggestion
program
  .option('--first')
  .option('-s, --separator <char>')
  .argument('<string>');
```

How about a line break to match the README?

Reply:
We run Prettier over the example files (but not over the source code in the README) and Prettier uses the compact layout for the short chaining calls in the simple example.

Reply:
I have wondered about whether should use the Prettier layout in the README, but I prefer being able to manually layout code and take extra space where I think it helps. Might just be habit though!

Reply:
I got it.